src/components/feedback/LinearProgress/LinearProgress.tsx

```diff
@@ -1,28 +1,37 @@
-import React from 'react'
+import React, { useMemo } from 'react'
 import PropTypes from 'prop-types'
 import Box from '@mui/material/Box'
 import { LinearProgress as MuiLinearProgress, Label } from './LinearProgressStyles'
 import { isNil } from 'ramda'
-import { LinearProgressProps, ProgressColor } from './types'
+import { LinearProgressProps } from './types'
+import { SxProps, Theme } from '@mui/system'
 
 /**
  * Progress indicators inform users about the status of ongoing processes, such as loading an app, submitting a form, or saving updates.
  */
 const LinearProgress: React.FC<LinearProgressProps> = ({
-  color = 'grey' as ProgressColor,
+  color = 'grey',
   variant,
   value,
   valueBuffer,
   showLabel,
   labelProps,
+  global,
   ...rest
 }) => {
+  const globalStyles: SxProps<Theme> = useMemo(
+    () => (global ? { position: 'fixed', zIndex: 9999, top: 0, left: 0, right: 0, overflow: 'hidden' } : {}),
+    [global]
+  )
+
+  const hasLabel = useMemo(() => !global && showLabel && !isNil(value), [global, showLabel, value])
+
   return (
-    <Box sx={{ display: 'flex', alignItems: 'center' }}>
-      <Box sx={{ width: '100%', mr: 1 }}>
+    <Box sx={{ display: 'flex', alignItems: 'center', ...globalStyles }}>
```

Comment:
The recommended way to styled a Material-UI component or html tag is by using `styled` API which is recommended by emotion. We have plenty examples in this library.